### PR TITLE
fixed url setup for order form

### PR DESCRIPTION
**Pull request by stuart-bass-bjss**
[AB#6829](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6829)

### CODE REVIEW

File: local-helm/environments/local-docker.yaml [hunk 1]
```diff
@@ -306,38 +306,8 @@ of:
...
-      urlConfig:
-        name: "bc-buyingcatalogue"
-        key: redis-url 
-      port: "6379"
-      tls: false
-      passSecret:
-        name: "bc-buyingcatalogue"
-        key: "redis-password"
+    tls: []  
+  baseUri: "/order"
```
Comment (sam-roberts-bjss) [inline local-helm/environments/local-docker.yaml:310]: Would you be able to provide some feedback in regards to the removal of these settings? As I was thinking the order form will still need to know about the client_id etc...

**Review by sam-roberts-bjss — COMMENTED** [review]
I maybe missing something but I can't find any configuration for ordapi in the local-docker.yaml file. Will this be added as part of another PR?

File: local-helm/environments/local-docker.yaml (hunk 1 above)
Comment (stuart-bass-cgi) [reply local-helm/environments/local-docker.yaml:310]: so, these are duplicated between here and the main values file (higher level), so I removed them from here

Comment (sam-roberts-bjss) [reply local-helm/environments/local-docker.yaml:310]: 👍

**Review by sam-roberts-bjss — APPROVED** [review]

**Pull request merged**